modules/localgov_services_sublanding/src/Plugin/Field/FieldFormatter/LinkNodeReference.php

```diff
@@ -112,16 +116,33 @@ private function buildInternal(LinkItem $item, $langcode) {
       $entity_type = key($params);
       $entity = $this->entityTypeManager->getStorage($entity_type)->load($params[$entity_type]);
 
-      if ($entity) {
+      if ($entity and $entity->access('view')) {
```

Comment:
Nit-pick: (and I discussed this with Stephen, who prefers `and`) but core and coding standards prefer `&&` https://www.drupal.org/project/coding_standards/issues/2890524

Reply:
I am so glad that proposal never made it :D

Reply:
It did 'approved pending coder rule' :-P